smart-alluxio/src/main/java/org/smartdata/actions/alluxio/AlluxioCopyRunner.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.smartdata.actions.alluxio;
+
+import java.io.IOException;
+
+import org.smartdata.actions.hdfs.copy.CopyRunner;
```

Comment:
Sure, perhaps we may refactor it when SSM on Alluxio almost works.
In this case, anything about HDFS is in package org.smartdata.actions.hdfs, anything about Alluxio is in package org.smartdata.actions.alluxio, and for anything common, I prefer package org.smartdata.actions rather than org.smartdata.actions.common.